examples/shopping-cart/src/reducers/products.js

```diff
@@ -49,10 +49,8 @@ export default combineReducers({
   visibleIds
 })
 
-export function getProduct(state, id) {
-  return state.byId[id]
-}
+export const getProduct = (state, id) => state.byId[id]
 
-export function getVisibleProducts(state) {
-  return state.visibleIds.map(id => getProduct(state, id))
-}
+export const getVisibleProducts = state => state.visibleIds.map(id =>
```

Comment:
If the arrow body is multiline please always start the body on the next line.
Please make it like:

```js
export const getVisibleProducts = state =>
  ...
```

or

```js
export const getVisibleProducts = state => (
  ...
)
```